agent.py

```diff
@@ -822,10 +820,9 @@ def findTimeToLive(self, ageLimited=False):
 
     def findEmptyNeighborCells(self):
         emptyCells = []
-        neighborCells = self.cell.neighbors.values()
-        for neighborCell in neighborCells:
-            if neighborCell.agent == None:
-                emptyCells.append(neighborCell)
+        for cell in  self.cell.neighbors.values():
```

Comment:
On second thought, this change should be reverted. It doesn't gain us anything and incurs an extra function call without added clarity for what's happening.